libs/openFrameworks/utils/ofSystemUtils.cpp

```diff
@@ -753,131 +730,6 @@ string ofSystemTextBoxDialog(string question, string text){
 		 DestroyWindow(dialog);
 		 EnableWindow(WindowFromDC(wglGetCurrentDC()), TRUE);
 
-	#else
-
```

Comment:
It's even says in comment:

```
//we have to do this because mingw wants non wide strings and vs wants wide strings
```

But it's not true. MinGW wants non wide strings when `UNICODE` and `_UNICODE` are not defined, if you define them it perfectly works with the code from `#ifdef _MSC_VER` block because it's the same Windows calls except it uses wide strings.